particles/Tutorial/Kotlin/Demo/TTTGameData.arcs

```diff
@@ -0,0 +1,44 @@
+import 'TTTGameSchemas.arcs'
+
+resource ComputerDefault
```

Comment:
FYI someday we'll make it better with: https://github.com/PolymerLabs/arcs/issues/3849